### PR TITLE
Support localized web UI

### PR DESCRIPTION
**Pull request by qwerty287**
Allow to translate the web UI. I added some docs, take a look at them for the instructions.
Only issue is date/time formatting: this is not easy and our formatters must import every language that's added. There's an formatter in vue-i18n, the used i18n lib (https://vue-i18n.intlify.dev/guide/essentials/datetime.html). I will try to migrate it to this.

address of #258

### CODE REVIEW

**Comment by 6543** [comment]
@qwerty287 please resolve :)

**Comment by 6543** [comment]
I cant find a menue to select languages, are you able to add it wothin this pull?

**Comment by anbraten** [comment]
> I cant find a menue to select languages, are you able to add it wothin this pull?

It uses your browser locale.

**Comment by 6543** [comment]
Oh, as default this is good, i still think you should be able to change it

File: docs/docs/92-development/07-translations.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Translations
+
+You can easily translate Woodpecker into your language. Therefore, copy the file `web/src/assets/locales/en.json` to the same path with your language's code and `.json` as name.
```
Comment (6543) [inline docs/docs/92-development/07-translations.md:3]: you should mention  https://vue-i18n.intlify.dev/ somewhere here

**Review by 6543 — APPROVED** [review]
updated #258 so we can merge this :)

**Pull request merged**